build/build.py

```diff
@@ -11,6 +11,19 @@
 import tarfile
 import zipfile
 
+__version = "0.56.0.0"
+
+def __globalVariables( buildDir ) :
+
+	return {
+		"buildDir" : buildDir,
+		"jobs" : multiprocessing.cpu_count(),
+		"path" : os.environ["PATH"],
+		"version" : __version,
+		"platform" : "osx" if sys.platform == "darwin" else "linux",
```

Comment:
Can we make this windows friendly to save Eric having to come back to it?

Reply:
There's a lot more work than this to make this project windows-friendly, and although it's a trivial change I'm sure, I don't have any way of testing...